src/org/opensolaris/opengrok/analysis/AnalyzerGuru.java

```diff
@@ -125,7 +125,14 @@
      * Define very close to OPENING_MAX_CHARS so the input stream stays easily
      * in the <code>mark</code> limit
```

Comment:
given this bug this comment is not very relevant. What about unifying OPENING_MAX_CHARS with OPENING_BUF_SIZE ?

Reply:
You're right. It's not useful as a distinct value. I pushed a removal.